reference-architecture/aws-ansible/playbooks/roles/prerequisites/tasks/main.yaml

```diff
@@ -11,3 +11,4 @@
       - bash-completion
       - docker
       - python-httplib2
+  when: ansible_pkg_mgr != "unknown" 
```

Comment:
Without it doing an install on Atomic "Could not find a module for unknown" so this way we can bypass if host is Atomic